src/QmlControls/CameraSettings.qml

```diff
@@ -0,0 +1,326 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.14
+import QtPositioning 5.14
+
+Rectangle {
+    id: root
+    color: '#000000'   // black
+
+    property var rootWidth: null;
+    property var rootHeight: null;
+
+    width: rootWidth / 3.24
+    height: rootHeight / 1
+
+    Text {
+        id: cameraSettingsText
+        color: "#FFFFFF"    // white
+        text: qsTr("Camera Settings")
+        //font.pixelSize: 25
+        font.pixelSize: ScreenTools.defaultFontPixelHeight * 2.5
+        font.bold: true
+        font.family: 'Arial'
+        anchors.top: parent.top
+        anchors.left: parent.left
+        anchors.leftMargin: 10
+        anchors.topMargin: 12  // Add margin from top of the parent if needed
+    }
+
+    Column {
+        id: column1
+        anchors.top: text1.bottom
+        anchors.topMargin: 15 // Editable margin between text1 and column1
+        anchors.left: parent.left
+        anchors.right: parent.right
+        anchors.bottom: parent.bottom
+
+        spacing: 4
+
+
+        ItemDelegate {
+            id: itemDelegate
+            width: parent.width
+            height: 40
+
+            Text {
+                anchors.left: parent.left
+                anchors.leftMargin: 15
+                text: 'Grid'
+                //font.pixelSize: 20
+                font.pixelSize: ScreenTools.defaultFontPixelHeight * 2
+                color: "#FFFFFF"
+                // verticalAlignment: Text.AlignVCenter
+            }
+        }
+
+        ToolSeparator {
+            width: parent.width
+            height: 13
+            orientation: Qt.Horizontal
+        }
+
+        ItemDelegate {
+            id: itemDelegate1
+            width: parent.width
+            height: 40
+            enabled: true
+            hoverEnabled: true
+
+            Text {
+                anchors.left: parent.left
+                anchors.leftMargin: 15
+                color: "#ffffff"
+                text: qsTr("Center Point")
+                font.pixelSize: ScreenTools.defaultFontPixelHeight * 2
+            }
+        }
+
+        ToolSeparator {
+            width: parent.width
+            height: 13
+            orientation: Qt.Horizontal
+        }
+
+        SwitchDelegate {
+            id: switchDelegate
```

Comment:
The IDs for ItemDelegate and SwitchDelegate components are unique but redundant if not referenced elsewhere. Remove unused IDs to reduce clutter.